<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExit"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionFont"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Document</string>
   </property>
   <property name="toolTip">
    <string>tworzy nowy plik</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="folder-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save_as</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset theme="document-print">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="mail-send-receive">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset theme="insert-text">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
